src/policy_downloader.rs

```diff
@@ -245,7 +247,20 @@ mod tests {
         // of the unit tests
         static ref DOWNLOADER: Mutex<Downloader> = Mutex::new({
             let rt = Runtime::new().unwrap();
-            rt.block_on(async { Downloader::new(None, None, true, None).await.unwrap() })
+            let docker_config_path = BaseDirs::new().map(|bd|
+                bd.home_dir().join(".docker").join("config.json")
+            );
+            let docker_config = if let Some(dcp) = docker_config_path {
+                if dcp.exists() {
+                    Some(read_docker_config_json_file(&dcp).expect("Error reading docker config file"))
+                } else {
+                    None
+                }
+            } else {
+                None
+            };
+            //let docker_config = read_docker_config_json_file(path: &Path)
```

Comment:
I just noticed this leftover, I'll remove it once the 1st run of unit test completes. I just want to see the results :crossed_fingers: